base/src/main/java/com/smartdevicelink/proxy/rpc/WindowStatus.java

```diff
@@ -0,0 +1,54 @@
+package com.smartdevicelink.proxy.rpc;
+
+import android.support.annotation.NonNull;
+
+import com.smartdevicelink.proxy.RPCStruct;
+
+import java.util.Hashtable;
+
```

Comment:
```suggestion
/**
 * Describes the status of a window of a door/liftgate etc.
 *
 * <p><b>Parameter List</b></p>
 *
 * <table border="1" rules="all">
 *  <tr>
 *      <th>Param Name</th>
 *      <th>Type</th>
 *      <th>Description</th>
 *      <th>Required</th>
 *      <th>Version Available</th>
 *  </tr>
 *  <tr>
 *      <td>location</td>
 *      <td>Grid</td>
 *      <td></td>
 *      <td>Y</td>
 *      <td></td>
 *  </tr>
 *  <tr>
 *      <td>state</td>
 *      <td>WindowState</td>
 *      <td></td>
 *      <td>Y</td>
 *      <td></td>
 *  </tr>
 * </table>
 *
 * @since SmartDeviceLink 7.0.0
 */
```
Please use the [RPC generator](https://github.com/smartdevicelink/sdl_java_suite/tree/master/generator) in the sdl_java_suite project to ensure the RPC matched the spec exactly. The generator will also update the rest of the JavaDocs in this file to ensure everything matches the RPC Spec exactly